#message containing mixer ouput values which are to be given to JSBsim for simulation
#vehicle is the quadshot
float32 aileron
float32 elevator
float32 throttle_0	#front right engine
float32 throttle_1	#rear left engine
float32 throttle_2  #front left engine
float32 throttle_3  #rear right engine

